File: RPG_Client/Assets/Scripts/Common/MsgProtocol.cs
using UnityEngine;
using System.Collections;
/// <summary>
/// 定义消息类型
/// </summary>
public class MsgProtocol {
    //请求协议后，服务端返回状态
    public const int Error = -1;
    public const int Success = 0;

    //协议类型
    public const int Login = 1;
    public const int Get_ServerList = 2;
    public const int Register = 3;
    public const int Query_Status = 4;
    public const int PreSelectHero = 5;
	public const int EnterGame = 6;
	public const int Get_TaskList = 7;
    public const int Update_PlayerInfo = 8;
    public const int Get_EquipList = 9;
	public const int Dress_Equip = 10;
	public const int Sale_Equip = 11;
    public const int Upgrade_Equip = 12;
    public const int Undress_Equip = 13;
	public const int AcceptTask = 14;
	
	public const int Get_SkillList = 15;
	public const int Upgrade_Skill = 16;

    public const int Get_PlayerInfo = 17;
}

File: RPG_Server/JWorkSpace/JServer/src/jg/rpg/common/protocol/MsgProtocol.java
package jg.rpg.common.protocol;
/**
 * 协议描述类，仅仅起标识作用
 */
public class MsgProtocol {
	public final static int  Error = -1;
	public final static int  Success = 0;
	
	
	public final static int  Login = 1;
	public final static int Get_ServerList = 2;
	public final static int Register = 3;
	public final static int Query_Status = 4;
	public final static int PreSelectHero = 5;
	public final static int EnterGame = 6;
	public final static int Get_TaskList = 7;
	public final static int Update_PlayerInfo = 8;
	
	public final static int Get_EquipList = 9;
	public final static int Dress_Equip = 10;
	public final static int Sale_Equip = 11;
	public final static int Upgrade_Equip = 12;
	public final static int Undress_Equip = 13;
	public final static int AcceptTask = 14;
	
	public final static int Get_SkillList = 15;
	public final static int Upgrade_Skill = 16;
	
	public final static int Get_PlayerInfo = 17;
	
}
